config/app.js

```diff
@@ -3,6 +3,10 @@ const { joinPath } = require('../lib/utils')
 module.exports = {
   title: 'ZWave To MQTT',
   storeDir: process.env.STORE_DIR || joinPath(true, 'store'),
+  defaultUser: 'admin',
+  defaultPsw: 'zwave',
+  sessionSecret:
+    process.env.SESSION_SECRET || 'G1FoTIfjhsPO80MpJSUx9oaDDHueTkW1',
```

Comment:
this feels like a bad idea?

Reply:
Alternatives?

Reply:
if you _really_ need to set it, make it more obvious its the default one, for example 
```suggestion
    process.env.SESSION_SECRET || 'DEFAULT_SESSION_SECRET_CHANGE_ME',
```
or if auth is enabled, throw an exception if its undefined with advice on how to generate one and how to define it < preferred option

Reply:
done